test/e2e/common/node/helpers/resize.go

```diff
@@ -535,3 +536,35 @@ func formatErrors(err error) error {
 	}
 	return fmt.Errorf("[\n%s\n]", strings.Join(errStrings, ",\n"))
 }
+
+// TODO: Remove the rounded cpu limit values when https://github.com/opencontainers/runc/issues/4622
+// is fixed.
+func GetCPULimitCgroupExpectations(cpuLimit *resource.Quantity) []string {
```

Comment:
Note to reviewers: this is moved here temporarily to resolve a circular dependency. This will be split into a separate file in the same package in https://github.com/kubernetes/kubernetes/pull/127192
/cc @hshiina